Clean/Types/U32.lean

```diff
@@ -44,6 +47,12 @@ def is_normalized (x: U32 (F p)) :=
 def value (x: U32 (F p)) :=
   x.x0.val + x.x1.val * 256 + x.x2.val * 256^2 + x.x3.val * 256^3
 
+/--
+Return the value of a 32-bit unsigned integer as a field
```

Comment:
There is a missing word in this comment:

`Return the value of a 32-bit unsigned integer as a field **element.**`

Admittedly, it is a non-issue, but we might still want to fix it for the sake of clarity.

Reply:
ah, yep I'm actually used to just saying "field" instead of "field element", because this was the widely used shorthand in the ecosystem I worked in (Mina / o1js).
iIn Starkware/Cairo they say "felts" :D)